src/common/cast/remote-control.js

```diff
@@ -216,10 +217,14 @@ function reconstructPlayerComponents(snapshot: PlayerSnapshot): void {
 
 function configurePlayback(playbackConfig: Object): void {
   const {autoplay, startTime} = playbackConfig;
+  const storageConfig = StorageManager.getStorageConfig();
+  const {muted, volume} = storageConfig;
```

Comment:
what happens if muted and/or volume are undefined from storage manger?

Reply:
nothing will happens